Firebase/DynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v3.4.0
+- Bug fixes and internal SDK changes. (#2238, #2220)
```

Comment:
It would be good to change this on master to be more descriptive ("Bug fixes")

Reply:
@dmandar FYI